js/constants/constants.js

```diff
@@ -7,7 +7,8 @@ module.exports = {
     PLAYING : "playing",
     PAUSE : "pause",
     END : "end",
-    ERROR : "error"
+    ERROR : "error",
+    TOUCH: "touch"
```

Comment:
I think we may need this to be TOUCHING or maybe we can think of a better name. Because the point is that you are in a state where the camera is changing direction, right?